### ISSUE TITLE
pytest 5.4.0 compatibility

### ISSUE THREAD

**Issue by Xelia** [pytest-dev/pytest-factoryboy#89]
Tests failing

```
============================================================================================================ ERRORS =============================================================================================================
________________________________________________________________________________________ ERROR collecting tests/test_factory_fixtures.py ________________________________________________________________________________________
In function "test_parametrized":
Parameter "edition__year" should be declared explicitly via indirect or in function itself
__________________________________________________________________________________________ ERROR collecting tests/test_lazy_fixture.py __________________________________________________________________________________________
In function "test_lazy_attribute":
Parameter "user__password" should be declared explicitly via indirect or in function itself
______________________________________________________________________________________ ERROR collecting tests/test_postgen_dependencies.py ______________________________________________________________________________________
In function "test_depends_on":
Parameter "foo__expected" should be declared explicitly via indirect or in function itself

```

**Comment by youtux** [comment]
I reported the issue at https://github.com/pytest-dev/pytest/issues/6909, let's see how we are going to move from there.

**Comment by youtux** [comment]
Fixed by pytest 5.4.1
